Source/Readers/ImageReader/ImageDataDeserializer.h

```diff
@@ -271,7 +271,7 @@ class ImageDataDeserializer : public DataDeserializerBase
         errno = 0;
         description.m_classId = strtoull(label.c_str(), &eptr, 10);
         if (label.c_str() == eptr || errno == ERANGE)
-            RuntimeError("Cannot parse label value on line %" PRIu64 ", second column, in file %s.", lineIndex, mapPath.c_str());
+            RuntimeError("Invalid map file format, must contain 2 or 3 tab-delimited columns, line %" PRIu64 " in file %s.", lineIndex, mapPath.c_str());
```

Comment:
The lines you cited refer to the classification case. Regression can indeed be any dimension! The reason I had to change the message in this line was because the unit tests expected to see that exact wording in the message. The other variant is just as fine in my mind, but then the unit test would have to be adapted.